data/dock.gschema.xml

```diff
@@ -27,4 +27,51 @@
       <description>An ordered array of app id's to show as launchers</description>
     </key>
   </schema>
+  <schema path="/io/elementary/dock/keybindings/" id="io.elementary.dock.keybindings">
+    <key type="as" name="launch-dock-1">
+      <default><![CDATA[['<Super>1']]]></default>
+      <summary>Launch the first dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-2">
+      <default><![CDATA[['<Super>2']]]></default>
+      <summary>Launch the second dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-3">
+      <default><![CDATA[['<Super>3']]]></default>
+      <summary>Launch the third dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-4">
+      <default><![CDATA[['<Super>4']]]></default>
+      <summary>Launch the fourth dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-5">
+      <default><![CDATA[['<Super>5']]]></default>
+      <summary>Launch the fith dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-6">
+      <default><![CDATA[['<Super>6']]]></default>
+      <summary>Launch the sixth dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-7">
+      <default><![CDATA[['<Super>7']]]></default>
+      <summary>Launch the seventh dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-8">
+      <default><![CDATA[['<Super>8']]]></default>
+      <summary>Launch the eigth dock item</summary>
+      <description></description>
+    </key>
+    <key type="as" name="launch-dock-9">
+      <default><![CDATA[['<Super>9']]]></default>
+      <summary>Launch the nineth dock item</summary>
+      <description></description>
+    </key>
```

Comment:
Perhaps we should only allow configuring the modifiers and always assume 1-9? We've had https://github.com/elementary/switchboard-plug-keyboard/issues/153 open for quite a while with this same issue that it makes changing this shortcut painful if you have to change nine when really you probably just want to choose between some combination of Super, Shift, Alt, and Ctrl

Reply:
I think a valid use case for changing the 1-9 is if you want f1 to f9 for this and 1-9 for workspaces or the other way round or maybe use the numpad or 1-5 for apps and 6-9 for workspaces? Maybe we could leave it like this here and handle auto changing all of them in the keyboard plug so that we still have the option for 1. multiple shortcuts for the same thing and 2. different keys?

Reply:
Ah yeah that's valid. I hadn't considered using something other than number keys. Good point!